### PR TITLE
Feat(arbitrum): Arbitrum bridge

### PR DESCRIPTION
**Pull request by Quazia**
Still needs some more testing - will edit and give a more complete PR description after I wrap up testing. There's a lot of brittle approaches here, Arbitrum doesn't give us a ton of assistance in terms of an API etc and the way they handle bridges unfolds across what are essentially three different approaches. I documented some of the process in the readme with links to example transactions.

### CODE REVIEW

File: packages/arbitrum/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// This file is standard for more projects.
+// The main degree of nuance is in the
```
Comment (jimobrien) [inline packages/arbitrum/src/index.ts:2]: `is in the` 

I'm waiting in suspense!

File: packages/connext/package.json [hunk 2]
```diff
@@ -15,9 +15,9 @@
     "bench": "vitest bench",
     "bench:ci": "CI=true vitest bench",
     "build": "pnpm run clean && pnpm run build:cjs && pnpm run build:esm && pnpm run build:types",
-    "build:cjs": "tsc --project tsconfig.build.json --module commonjs --outDir ./dist/cjs --removeComments --verbatimModuleSyntax false && echo > ./dist/cjs/package.json '{\"type\":\"commonjs\"}'",
-    "build:esm": "tsc --project tsconfig.build.json --module es2015 --outDir ./dist/esm && echo > ./dist/esm/package.json '{\"type\":\"module\",\"sideEffects\":false}'",
-    "build:types": "tsc --project tsconfig.build.json --module esnext --declarationDir ./dist/types --emitDeclarationOnly --declaration --declarationMap",
+    "build:cjs": "tsc --project tsconfig.build.json --module commonjs --moduleResolution node --outDir ./dist/cjs --removeComments --verbatimModuleSyntax false && echo > ./dist/cjs/package.json '{\"type\":\"commonjs\"}'",
```
Comment (jimobrien) [inline packages/connext/package.json:18]: reminder that we can remove these once #23 is merged

File: packages/arbitrum/src/chain-ids.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// The arbitrum Native Bridge only supports three chains, Arbitrum, Arbitrum Nova, and Ethereum
+export const CHAIN_ID_ARRAY = [1, 42161, 42170]
```
Comment (jimobrien) [inline packages/arbitrum/src/chain-ids.ts:2]: any reason not to just use the named variables in the list? 

`export const CHAIN_ID_ARRAY = [ETH_CHAIN_ID, ARB_ONE_CHAIN_ID, ARB_NOVA_CHAIN_ID]`

**Review by jimobrien — APPROVED** [review]
LGTM

File: packages/arbitrum/src/chain-ids.ts (hunk 3 above)
Comment (Quazia) [reply packages/arbitrum/src/chain-ids.ts:2]: It should be - good catch

**Pull request merged**